### DISCUSSION TITLE
Neovim 配置问题

### DISCUSSION THREAD

**Discussion by BitterDarker** [HUAHUAI23/nvim64#7] [Q&A]
麻烦大佬讲解一下 操作一下这些
tar -xf nvim.tar
cd nvim64
bash initial.sh
rm -rf trush
cd nvim-linux64/bin/

# 将为当前shell环境添加alias，即终端输入n就可以打开neovim，等效于nvim-linux64/bin/nvim
source int42.sh第一次进入neovim后，neovim会自动下载插件

现在是没有自动下载插件

**Comment by BitterDarker** [comment]
darker@Darker:~/tools/nvim64$ bash initial.sh
安装已经完成
进入nvim-linux64/bin目录执行 source int42.sh
执行完后当前shell环境将增加alias n，该alias会打开neovim即在终端输入n即可
第一次进入neovim会下载插件，请确保访问GitHub的网络环境
插件的下载需要git命令，请确保有git命令，除此之外确保有nodejs
第二次进入neovim请输入命令:PackSync 和 :TSupdate
输入命令:checkhealth检查neovim的健康状态
可以将trush目录删掉
darker@Darker:~/tools/nvim64$ rm -rf trush
darker@Darker:~/tools/nvim64$ cd nvim-linux64/bin/
darker@Darker:~/tools/nvim64/nvim-linux64/bin$ source int42.sh
darker@Darker:~/tools/nvim64/nvim-linux64/bin$ nvim
darker@Darker:~/tools/nvim64/nvim-linux64/bin$ nvim
darker@Darker:~/tools/nvim64/nvim-linux64/bin$

**Comment by HUAHUAI23** [comment]
没下载插件 很可能是网络原因，一般情况下访问 github 会很慢，建议翻墙试一下
